src/screens/ChannelListScreen/elements/Search.js

```diff
@@ -10,7 +10,8 @@ import MemoIcSearch from '../../../assets/icons/Ic_search';
 import StringConstant from '../../../utils/string/StringConstant';
 import dimen from '../../../utils/dimen';
 import {DISCOVERY_TAB_USERS} from '../../../utils/constants';
-import {COLORS, SIZES} from '../../../utils/theme';
+import {SIZES} from '../../../utils/theme';
+import {colors} from '../../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for colors has been updated. Ensure that the new `colors` module provides all the necessary color constants that were previously available in `COLORS`.
```diff
- import {COLORS, SIZES} from '../../../utils/theme';
+ import {SIZES} from '../../../utils/theme';
+ import {colors} from '../../../utils/colors';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->